### PR TITLE
IGAPP-702: Enable screens

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

Another workaround still needed :/ But not a bad one I guess...

:exclamation: PLEASE TEST THIS WITH A PRODUCTION BUILD! :exclamation:

### CODE REVIEW

File: native/src/components/RemoteContent.tsx [hunk 1]
```diff
@@ -37,20 +37,25 @@ type PropType = {
...
+  const defaultWebviewHeight = 1
+  const [webViewHeight, setWebViewHeight] = useState(defaultWebviewHeight)
+  const [webViewWidth, setWebViewWidth] = useState(defaultWebviewHeight)
+
   useEffect(() => {
-    if (webViewHeight !== 0) {
+    if (webViewHeight !== defaultWebviewHeight) {
       onLoad()
     }
   }, [onLoad, webViewHeight])
```
Comment (f1sh1918) [inline native/src/components/RemoteContent.tsx:49]: why do we have to watch for changes of the onLoad fkt?

Comment (klinzo) [reply native/src/components/RemoteContent.tsx:49]: If the onLoad func changes for whatever reason, you want to execute it again to apply those changes, In practice, this probably won't happen often though. There is also a recommended eslint rule forcing us to use exhaustive dependencies, i.e. specifying all dependencies of hooks.

Comment (maxammann) [reply native/src/components/RemoteContent.tsx:49]: Its better to add whatever could change, than thinking about it :D

**Review by maxammann — APPROVED** [review]
Looks good, I'll test this after lunch

**Comment by maxammann** [comment]
navigating to webviews works for me :) :+1:

**Review by Taggotty — APPROVED** [review]
Tested

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**